codalab/lib/worksheet_util.py

```diff
@@ -992,7 +994,9 @@ def flush_worksheets():
             worksheet_infos[:] = []
             blocks.append(
                 MarkupBlockSchema()
-                .load({'text': 'Error on line %d: %s' % (raw_index, str(e))})
```

Comment:
You have to update `create_sample_worksheet.py` as well because we rely on regex to match against this error message.